manifests/config.pp

```diff
@@ -1,18 +1,11 @@
-# == Class nomad::config
-#
 # This class is called from nomad::init to install the config file.
 #
-# == Parameters
-#
-# [*config_hash*]
-#   Hash for nomad to be deployed as JSON
-#
-# [*purge*]
-#   Bool. If set will make puppet remove stale config files.
-#
+# @api private
+# @param config_hash Hash for nomad to be deployed as JSON.
+# @param purge If set will make puppet remove stale config files.
 class nomad::config (
-  $config_hash,
-  $purge = true,
+  Hash $config_hash,
+  Boolean $purge = true,
```

Comment:
Given a lot of the variables here already use `$nomad::VAR`, should these be removed and use `$nomad::config_hash` instead? I opened https://github.com/voxpupuli/puppet-nomad/pull/29 and perhaps this should be rebased on that.

Reply:
i approved so lets get that merged and then i ll rebase

Reply:
Merged